lib/src/utils/instabug_navigator.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter/material.dart';
+import 'package:instabug_flutter/instabug_flutter.dart';
+import 'package:instabug_flutter/src/utils/instabug_logger.dart';
+import 'package:instabug_flutter/src/utils/repro_steps_constants.dart';
+import 'package:instabug_flutter/src/utils/screen_loading/screen_loading_manager.dart';
+import 'package:instabug_flutter/src/utils/screen_name_masker.dart';
+
+class InstabugNavigator {
+  final List<String> _steps = [];
+
+  void screenChanged(String? rawScreenName) {
+    try {
+      final screenName = rawScreenName == null || rawScreenName.isEmpty
+          ? ReproStepsConstants.emptyScreenFallback
+          : rawScreenName;
```

Comment:
I see this is changing the `null` behavior for screen names that are `null` to change to `N/A`, I believe this is a breaking change that we need to add to the changelog since such change will affect the grouping of screen info on the "Screens" page in the dashboard. What do you think?